add-on/src/lib/ipfs-client/reloaders/webUiReloader.js

```diff
@@ -0,0 +1,30 @@
+const { ReloaderBase } = require('./reloaderBase')
+
+class WebUiReloader extends ReloaderBase {
+  /**
+   * Performs url validation for the tab. If tab is a WebUI tab.
+   *
+   * @param {Object} tab
+   * @returns {boolean}
+   */
+  validation ({ url }) {
+    const bundled = !url.startsWith('http') && url.includes('/webui/index.html#/')
+    const ipns = url.includes('/webui.ipfs.io/#/')
+    return bundled || ipns
+  }
```

Comment:
This functionality was brought in from: https://github.com/ipfs/ipfs-companion/blob/main/add-on/src/lib/ipfs-client/index.js#L64-L68

But now that I think more about it, this will be handled in the `localGatewayReloader` check as `isIPFS.url || isIPFS.subdomain` returns true for `http://127.0.0.1:5001/ipfs/bafybeibozpulxtpv5nhfa2ue3dcjx23ndh3gwr5vwllk7ptoyfwnfjjr4q` hardcoded cid.